.circleci/config.yml

```diff
@@ -364,7 +364,7 @@ jobs:
           keys:
           - verdict-dependencies-{{ checksum "pom.xml" }}
       - run: pwd; ls;
-      - run: cd pyverdict; pip install pytest PyMySQL presto-python-client; python setup.py install;
+      - run: cd pyverdict; pip install pytest psycopg2 python-dateutil PyMySQL presto-python-client; python setup.py install;
```

Comment:
Since it looks like pyverdict requires python-dateutil, can you specify it in its dependency list in setup.py? Then, I think it should be automatically installed by "python setup.py install".

Reply:
Fixed.